gradle/dependencies.gradle

```diff
@@ -34,13 +34,13 @@ versions += [
   junit: "4.12",
   lz4: "1.3.0",
   metrics: "2.2.0",
-  powermock: "1.6.3",
+  powermock: "1.6.4",
   reflections: "0.9.10",
   rocksDB: "3.10.1",
   scalaTest: "2.2.5",
   scalaParserCombinators: "1.0.4",
-  slf4j: "1.7.6",
-  snappy: "1.1.2",
+  slf4j: "1.7.15",
+  snappy: "1.1.2.1",
   zkclient: "0.7",
   zookeeper: "3.4.6",
```

Comment:
Ah right. Linking the jira: [ZOOKEEPER-2347](https://issues.apache.org/jira/browse/ZOOKEEPER-2347)